WoWsShipBuilder.UI/UserControls/CaptainSkillSelector.axaml

```diff
@@ -19,23 +23,83 @@
     <Grid ColumnDefinitions="*">
 
         <DockPanel>
-            <Grid ColumnDefinitions="*,*" RowDefinitions="*,*" DockPanel.Dock="Top">
-                <StackPanel Name="CaptainPoints" HorizontalAlignment="Center" Orientation="Horizontal" Grid.Column="0" Grid.ColumnSpan="2" Grid.Row="0" Margin="0,5,0,0">                
-                    <TextBlock Text="{x:Static t:Translation.CaptainSkillSelector_Points}" FontSize="13" VerticalAlignment="Center" />
+            <Grid ColumnDefinitions="*,*,*" RowDefinitions="*,*" DockPanel.Dock="Top">
+                <StackPanel Name="CaptainPoints" Orientation="Horizontal" HorizontalAlignment="Center" Grid.Column="1" Grid.Row="0" Margin="0,5,0,5">                
+                    <TextBlock Text="{x:Static t:Translation.CaptainSkillSelector_Points}" FontSize="13" VerticalAlignment="Center"/>
                     <TextBlock Text="{Binding AssignedPoints}" Margin="5,0,0,0" FontSize="13" VerticalAlignment="Center" />
                     <TextBlock Text="/21" FontSize="13" VerticalAlignment="Center" />
                 </StackPanel>
-                <StackPanel HorizontalAlignment="Left" Orientation="Horizontal" Grid.Column="0" Grid.Row="1" Margin="0,5,0,0">
-                    <TextBlock Text="Captain:" Padding="5,0" VerticalAlignment="Center" />
-                    <ComboBox Items="{Binding CaptainList}" SelectedItem="{Binding SelectedCaptain}" VerticalAlignment="Center" >
+                <StackPanel HorizontalAlignment="Left" Orientation="Horizontal" Grid.Column="0" Grid.Row="1" Margin="0,5,0,5">
+                    <!--<TextBlock Text="Captain:" Padding="5,0" VerticalAlignment="Center" />-->
```

Comment:
Remove unnecessary code (no need to keep commented code, we have a VCS for that)